README.md

```diff
@@ -1,21 +1,27 @@
 # OpenIDConnect
 
-Client library for consuming and working with OpenID Connect Providers
+Firezone's fork of [DockYard's client library](https://github.com/dockyard/openid_connect) for consuming and working with OpenID Connect Providers.
 
-**[OpenIDConnect is built and maintained by DockYard, contact us for expert Elixir and Phoenix consulting](https://dockyard.com/phoenix-consulting)**.
+**[OpenIDConnect is originally built and maintained by DockYard, contact us for expert Elixir and Phoenix consulting](https://dockyard.com/phoenix-consulting)**.
+
+**This fork is maintained by the Firezone team**.
 
 ## Installation
 
-[Available in Hex](https://hex.pm/packages/openid_connect), the package can be installed as:
+~~[Available in Hex](https://hex.pm/packages/openid_connect), the package can be installed as:~~
+
+EDIT: This fork is *not* published to Hex (yet).
 
 Add `openid_connect` to your list of dependencies in `mix.exs`:
 
 ```elixir
 def deps do
-  [{:openid_connect, "~> 0.2.2"}]
+  [{:openid_connect, github: "firezone/openid_connect"}]
```

Comment:
I will `git checkout upstream/master -- README.md` to reset those changes if this PR is looking good enough to be merged.

Reply:
there's a lot to review, the high level proposals in the PR comment all make sense to me though. As far as validating the implementation I'll need time but I'd like to get these requests fixed before I review